app/services/claims/injection_channel.rb

```diff
@@ -0,0 +1,8 @@
+module Claims
+  class InjectionChannel
+    def self.for(claim)
+      return 'cccd_development' if claim.nil?
+      claim.agfs? ? 'cccd_ccr_injection' : 'cccd_cclf_injection'
```

Comment:
does it need to be `#channel_name`? - [the env var](https://github.com/ministryofjustice/advocate-defence-payments-deploy/pull/164) and old settings was - worth deleting that too

Reply:
No, the slack API documentation is unclear, but submitting messages with just the channel name, not #-prefixed, works fine